src/redux/actions/authAuctions.js

```diff
@@ -0,0 +1,23 @@
+export const signIn = userCredentials => (dispatch, getState, { getFirebase }) => {
```

Comment:
What is `getFirebase` here?

Reply:
getFirebase is a method that comes from 'react-redux-firebase' package and allows me access informations from firebase... to get data or to send data. in this case i was sending data to check the credentials for that specific user to see if there is an user with that email and password.